### PR TITLE
Declare connection for bindings N thru S

### PR DESCRIPTION
**Pull request by mlobstein**
References #14885

Declared connection type in addon.xml based on review of documentation/README.

### CODE REVIEW

**Review by renescherer — APPROVED** [review]

**Review by pacive — APPROVED** [review]

**Review by theater — APPROVED** [review]

File: bundles/org.openhab.binding.smhi/src/main/resources/OH-INF/addon/addon.xml [hunk 1]
```diff
@@ -6,5 +6,7 @@
 	<type>binding</type>
 	<name>SMHI Binding</name>
 	<description>Binding for getting weather forecasts from the Swedish Meteorological and Hydrological Institute (SMHI)</description>
+	<connection>cloud</connection>
+	<countries>se, no, dk, fi</countries>
```
Comment (jlaur) [inline bundles/org.openhab.binding.smhi/src/main/resources/OH-INF/addon/addon.xml:10]:
```suggestion
	<countries>se,no,dk,fi</countries>
```

**Review by jlaur — CHANGES_REQUESTED** [review]
Thanks! Only one finding as I don't think spaces are allowed within the `countries` value:

```xml
<xs:simpleType name="countryType">
  <xs:restriction base="xs:string">
    <xs:pattern value="[a-z]{2}(,[a-z]{2})*"/>
  </xs:restriction>
</xs:simpleType>
```

File: bundles/org.openhab.binding.smhi/src/main/resources/OH-INF/addon/addon.xml (hunk 1 above)
Comment (mlobstein) [reply bundles/org.openhab.binding.smhi/src/main/resources/OH-INF/addon/addon.xml]: fixed, thanks.

**Review by jlaur — APPROVED** [review]
LGTM

**Pull request merged**